rero_ils/modules/cli.py

```diff
@@ -1352,3 +1352,80 @@ def dump_es_mappings(verbose, outfile):
             if outfile:
                 json.dump(mapping, outfile, indent=2)
                 outfile.write('\n')
+
+
+@utils.command('export')
+@click.option('-v', '--verbose', 'verbose', is_flag=True, default=False)
+@click.option('-p', '--pid_type', 'pid_type', default='doc')
+@click.option('-o', '--outfile', 'outfile', required=True,
```

Comment:
What do you think about having a default value as `doc_export.json` or something like `document_dump.json` or `20200518_doc_export.json`?
It's not important, just an idea :smiley: